core/src/test/java/io/grpc/internal/RetriableStreamTest.java

```diff
@@ -217,6 +217,15 @@ public void retry_everythingDrained() {
     ClientStream mockStream3 = mock(ClientStream.class);
 
     doReturn(mockStream1).when(retriableStreamRecorder).newSubstream(0);
+    doAnswer(new Answer<Void>() {
+        @Override
+        public Void answer(InvocationOnMock in) {
+          InsightBuilder insight = (InsightBuilder) in.getArguments()[0];
+          insight.appendKeyValue("server_addr", "2.2.2.2:443");
```

Comment:
existing terminology in our code I think points to this being a "backend_addr", not a server_addr.   (or possibly remote_addr)